Sources/OutlineView/OutlineViewDragAndDrop.swift

```diff
@@ -0,0 +1,139 @@
+import Foundation
+import AppKit
+
+/// A protocol for use with `OutlineView`, implemented by an object (most likely
+/// the data source) to handle receiving items that are dragged into the `OutlineView`.
```

Comment:
I would refrain from making any suggestions about the conforming type, as it is a stylistic choice left up to the developers and can vary vastly from one codebase to another. So, no "object", or "most likely the data source".

Might I suggest:
> A delegate to interact with drop operations in an OutlineView.

Reply:
Good call. Fixed here https://github.com/Sameesunkaria/OutlineView/pull/10/commits/9a66eb8d6576eb16ca2d046bf17d013ac32249e3